docs/guides/python/ibis.md

```diff
@@ -3,147 +3,706 @@ layout: docu
 title: DuckDB with Ibis
 ---
 
-[Ibis](https://ibis-project.org/) is a Python library that allows queries to be written in a pythonic relational style and then be compiled into SQL.
-Ibis supports multiple database backends, including [DuckDB](https://ibis-project.org/backends/DuckDB/) by using [DuckDB's SQLAlchemy driver](https://github.com/Mause/duckdb_engine). Ibis expressions can also be combined with SQL statements.
+[Ibis](https://ibis-project.org) is a Python dataframe library that supports 15+ backends, with DuckDB as the default. Ibis with DuckDB provides a Pythonic interface for SQL with great performance.
 
 ## Installation
 
-To install only the DuckDB backend for Ibis, use the commands below. See the [Ibis DuckDB installation instructions](https://ibis-project.org/backends/DuckDB/) for a conda alternative. Note that DuckDB support was added in Ibis version 3.0.0.
-```python
-pip install 'ibis-framework[duckdb]' # duckdb, sqlalchemy, duckdb_engine and more are installed as dependencies
+You can pip install Ibis with the DuckDB backend:
+
+```bash
+pip install 'ibis-framework[duckdb]'
 ```
 
-## Querying DuckDB with Ibis
+or use conda:
+
+```bash
+conda install ibis-framework
+```
+
+or use mamba:
+
+```bash
+mamba install ibis-framework
+```
 
-The following example is loosely borrowed from the [Introduction to Ibis tutorial](https://nbviewer.org/github/ibis-project/ibis-examples/blob/main/tutorial/01-Introduction-to-Ibis.ipynb), which uses SQLite.
-First, we import Ibis, set it to interactive mode (just for demo purposes - it is faster to not use this option!), and then connect to an in-memory DuckDB instance. We can then inspect the tables in our database. 
+## Create a database file
```

Comment:
Please use headline capitalization for titles (just changed all other pages to follow this in bc73472c735e2425d0e7524666b62d406c218ec6 and other commits).